x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_alerts/alert_status.cy.ts

```diff
@@ -228,4 +239,29 @@ describe('Changing alert status', { tags: ['@ess', '@serverless'] }, () => {
         });
     });
   });
+
+  context.only('User is readonly', () => {
+    beforeEach(() => {
+      login();
+      visit(ALERTS_URL);
+      deleteAlertsAndRules();
+      createRule(getNewRule());
+      login(ROLES.reader);
+      visit(ALERTS_URL, { role: ROLES.reader });
+      waitForAlertsToPopulate();
+    });
+
+    it('should not allow users to change a single alert status', () => {
+      cy.get(TIMELINE_CONTEXT_MENU_BTN).should('not.exist');
```

Comment:
@michaelolo24 

This will remove complete context menu btn? Are other actions such as `Run Osquery` , `Add to Case`, etc are also not allowed?

Reply:
I need to update the role used. The `Roles.reader` affects those as well. Wanted to avoid creating another user, but maybe it's worth it... 🤷🏾‍♂️

Reply:
Once this is merged - https://github.com/elastic/kibana/pull/169017 - you can test with the appropriate roles!

@maximpn

Reply:
@yctercero looking at it now. Reader may still be the most accurate